src/storage/src/hummock/file_cache/meta.rs

```diff
@@ -127,6 +129,7 @@ where
                 fd,
                 0,
             )? as *mut u8;
+            madvise(ptr as *mut c_void, size, MmapAdvise::MADV_WILLNEED)?;
```

Comment:
need some comments for `madvise`, and i think we dont need to handle the result of `madvise` .  

IMO, we also can open the meta file although `madvise` fail. Hence, add some log to track it.